platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -58,30 +148,27 @@ NS_ASSUME_NONNULL_BEGIN
 
  @param mapView The map view whose viewpoint has changed.
  @param animated Whether the change caused an animated effect on the map.
+
+ @note If `-mapView:regionDidChangeWithReason:animated:` is implemented this method will not be called.
```

Comment:
We should remove mention of the `:nodoc:` reasonful method from public method docs.